core/src/main/scala/org/http4s/rho/PathBuilder.scala

```diff
@@ -26,14 +27,12 @@ final class PathBuilder[T <: HList](val method: Method, val path: PathRule)
 {
   type HeaderAppendResult[T <: HList] = Router[T]
 
-  override def headers: HeaderRule = EmptyHeaderRule
-
-  override def query: QueryRule = EmptyQuery
+  override val rules: RequestRule = EmptyRule
```

Comment:
For all the builder types, internally they now use a single set of rules (called `rules`...) instead of two different sets, `headers` and `query`.